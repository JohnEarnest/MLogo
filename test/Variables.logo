# Variables
#
# test the creation of local and global variables
# as well as reading from them.

local 'foobar 43
print :foobar

local 'baz [A C D C]
print :baz
print thing 'baz

make 'quux 967
print :quux