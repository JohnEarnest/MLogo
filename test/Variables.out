43
[A C D C]
[A C D C]
967
